lib/src/main/java/com/thenewboston/utils/PaginationOptions.kt

```diff
@@ -0,0 +1,30 @@
+package com.thenewboston.utils
+
+data class PaginationOptions(
+    val offset: Int?,
+    val limit: Int?
+) {
+    fun toQuery(): String {
+        return when {
+            offset == null || limit == null -> "?offset=0&limit=20"
+            else -> "?offset=$offset&limit=$limit"
+        }
+    }
+}
+
+object PAGE {
```

Comment:
Please follow mixed case for naming objects and class names. This should be `Page`